js/apps/system/_admin/aardvark/APP/frontend/js/routers/router.js

```diff
@@ -45,6 +45,8 @@
       'store/:name': 'storeDetail',
       'graphs': 'graphManagement',
       'graphs/:name': 'showGraph',
+      'g6graphs': 'g6GraphManagement',
```

Comment:
Is `g6GraphManagement` a copy (and replacement?) of the original `graphManagement` view? 
In the last couple of weeks/months, we also introduced EnterpriseGraphs and did several commits at least in this single file (maybe more). 

In case `g6GraphManagement` view  is a replacement, we need to make sure that all changes which have been done to the original ones, also are getting applied to this view. And in that case we should also drop one of them as it will become almost impossible to maintain two almost identical views.

PRs: I've quickly found:
- https://github.com/arangodb/arangodb/pull/16394
- https://github.com/arangodb/arangodb/pull/16575
- https://github.com/arangodb/arangodb/pull/16603

Important: This PR list does not must include all fixed we did. To get a real view, we need to diff those files and compare every line.